UrbanMobility/Trip/doc/spec.md

```diff
@@ -0,0 +1,111 @@
+# gtfs:Trip
+
+## Description
+
+See [https://developers.google.com/transit/gtfs/reference/#tripstxt](https://developers.google.com/transit/gtfs/reference/#tripstxt)
+
+## Data Model
+
++ `id`: Entity id. 
+    + It shall be `urn:ngsi-ld:gtfs:Trip:<trip_identifier>` being `trip_identifier` a value that can be derived from GTFS `trip_id`. 
+
++ `type`: Entity type. 
+    + It shall be equal to `gtfs:Trip`.
+    
++ `dateCreated` : Entity's creation timestamp.
+  + Attribute type: [DateTime](https://schema.org/DateTime)
+  + Read-Only. Automatically generated. 
+ 
++ `dateModified`: Last update timestamp of this Entity.
+  + Attribute type: [DateTime](https://schema.org/DateTime)
+  + Read-Only. Automatically generated.
+  
+ + `headSign`: Same as GTFS `trip_headsign`.
+  + Attribute type: Property. [Text](https://schema.org/Text).
+  + Optional 
+  
++ `shortName`: Same as GTFS `trip_short_name`.
+  + Attribute type: Property. [Text](https://schema.org/Text).
+  + Optional
+    
++ `direction`: Same as GTFS `direction_id`.
+  + Attribute type: Property. [Text](https://schema.org/Text).
+  + Allowed Values: `"0"` and `"1"` as per GTFS `direction_id`. 
+  + Optional
+    
++ `block`: Same as GTFS `block_id`.
```

Comment:
if block is not modeled, i am not sure it make much sense to keep it

Reply:
yes, I can think on something better